hilti/toolchain/src/compiler/global-optimizer.cc

```diff
@@ -212,12 +325,72 @@ struct Visitor : hilti::visitor::PreOrder<bool, Visitor> {
 
                 break;
             }
+
+            case Stage::PRUNE_DECLS:
+                // Nothing.
+                break;
         }
 
         return false;
     }
 
-    GlobalOptimizer::Hooks* _hooks = nullptr;
+    result_t operator()(const operator_::function::Call& call, position_t p) {
+        if ( ! call.hasOp0() )
+            return false;
+
+        auto id = call.op0().tryAs<expression::ResolvedID>();
+
+        if ( ! id )
+            return false;
```

Comment:
This shouldn't ever happen right? let's trigger an internal error then to make sure we catch it if it does.